yaptide/admin/simulators.py

```diff
@@ -423,6 +417,7 @@ def derive_key(encryption_password: str = password, encryption_salt: str = salt)
 
 @run.command
 @click.option('--name', type=click.Choice([sim.name for sim in SimulatorType]))
+@click.option('--path', type=click.Path(file_okay=False))
```

Comment:
```suggestion
@click.option('--path', type=click.Path(file_okay=False, path_type=Path))
```

you can ask click to make conversion to path